sympy/simplify/tests/test_trigsimp.py

```diff
@@ -378,12 +380,8 @@ def valid(a, b):
     for a in (1, I, x, I*x, 1 + I):
         w = exp(a)
         eq = y*(w - 1/w)/(w + 1/w)
-        s = simplify(eq)
-        assert s == exptrigsimp(eq)
```

Comment:
Has this assert been removed?

Reply:
@Upabjojr As simplify was called in exptrigsimp, exptrigsimp was equivalent to simplify and did nothing more than it, and that test was checking it.